api/src/main/kotlin/team/backend/domain/entity/Job.kt

```diff
@@ -0,0 +1,30 @@
+package team.backend.domain.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import team.backend.infrastructure.BaseEntity
+
+@Entity
+@Table(name = "tbl_job")
+class Job(
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "job_id")
+    val id: Long? = null,
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "job_type", nullable = false, insertable = false, updatable = false)
+    val type: JobType? = null,
+): BaseEntity()
+
+enum class JobType {
```

Comment:
Table records and enum are 1:1 matching. did we need a `tbl_job` table?

Reply:
I don't know how enum and 1:1 matching relate to table needs. I think a job table is necessary. This has already been discussed in advance.